cdbot/cogs/fun.py

```diff
@@ -256,7 +258,20 @@ async def on_raw_reaction_add(self, raw_reaction: RawReactionActionEvent):
                     await connection.execute(
                         "DELETE FROM quotes WHERE quote_id = $1", reaction.message.id
                     )
+                reaction_users = set()
+                async for user in reaction.users():
+                    reaction_users.add(user)
```

Comment:
Why?

Reply:
It errors if you don't use async, and I tried it in a few ways, if there is a simpler way to achieve the same result please lmk.